File: TreePathSum/TreePathSum.java
import java.util.*;
import crio.ds.Tree.TreeNode;

class TreePathSum {
	public static void main(String [] args) {
		Scanner sc = new Scanner(System.in);
		TreeNode root  = TreeNode.readTreeReturnRoot(sc);
		int k = sc.nextInt();
		Long answer = Solution.CountTreePathSumToK(root, k);
		System.out.print(answer);
	}
}

File: TreePathSum/TreePathSum.py
from crio.ds.Tree.TreeNode import *
from Solution import countTreePathsThatSumsToK





def main():


    root = readTreeReturnRoot()
    k = int(input())
    print(countTreePathsThatSumsToK(root, k))

if __name__ == '__main__':
    import sys
    sys.setrecursionlimit(100000)
    main()
